content/integrate/redis-data-integration/ingest/installation/_index.md

```diff
@@ -88,6 +88,107 @@ Once the installation is finished, RDI is ready for use.
 
 {{<note>}}RDI gives you instructions to help you create secrets and create your pipeline.{{</note>}}
 
+## "Silent" installation
+
+You can use the
+[`redis-di install`]({{< relref "/integrate/redis-data-integration/ingest/reference/cli/redis-di-install" >}})
+command with the `--file` option (or the shorter version `-f`) to supply answers
+to the installer's questions automatically using properties from a
+[TOML](https://toml.io/en/) file:
+
+```bash
+redis-di install --file silent.toml
+```
+
+### Silent install example
+
+The following example shows the properties for a typical
+silent install configuration:
+
+```toml
+title = "RDI Silent Installer Config"
+
+scaffold = true
+deploy_directory = "/opt/rdi/config"
+
+# If you are *not* using an existing RDI database and you want
+# the installer to create one then remove this section
+# and uncomment the [rdi.cluster] section below.
+[rdi.database]
+host = "localhost"
+port = 12001
+username = "username"
+password = "password"
+use_existing_rdi = true
+ssl = true
+
+# Uncomment this section and remove the [rdi.database] section above
```

Comment:
if you want RDI to create the db you still need to provide the RDI password to use
not sure if you always have to specify use_exisiting_rdi=false

Reply:
I've added some text to say that you should set use_exisiting_rdi=false because I figure it won't do any harm. We can remove this in the future if it isn't necessary.